src/App.jsx

```diff
@@ -6,12 +6,34 @@ import TodoList from './TodoList'
 import AddTodoForm from './AddTodoForm'
 
 function App() {
-  
+  const [list, setNewTodo] = useState([
```

Comment:
It’s better to use names like list and setList for consistency, adding set to the second argument.

For example: todo and setTodo, or list and setList.

Additionally, since the value in question is an array, naming it setNewTodo can be misleading. The term newTodo suggests an individual object, while in reality, you are managing a list.